src/components/NewTweetPopover/index.jsx

```diff
@@ -23,25 +23,19 @@ const useStyles = createUseStyles({
 });
 
 function NewTweetPopover({ handleClose }) {
-  // State
+  const { username } = useUser();
+
   const [tweet, setTweet] = useState('');
   const [loading, setLoading] = useState(false);
   const [error, setError] = useState(undefined);
 
-  // Context
-  const { username } = useUser();
-
-  // Variables
   const characterCount = tweet.length;
-  const MAX_CHARACTERS = 30; // Constant convention is all caps
+  const MAX_CHARACTERS = 120; // Constant convention is all caps
```

Comment:
I forgot at some point we reduced this to 30, so changing it back to 120 even though it's outside of the scope of this 'feature'

Reply:
```suggestion
  const WARNING_CHARACTERS = 20; // Constant convention is all caps
```

I suggest we increase the WARNING_CHARACTERS to match the increased MAX_CHARACTERS

How can I reference that part of the code in this comment?

Reply:
I suggest we also increase the WARNING_CHARACTERS

How can I reference that part of the code in this comment?

Reply:
A few lines down we use the `WARNING_CHARACTERS` subtracted from the `MAX_CHARACTERS`

https://github.com/karsonkalt/CTD_React_App/blob/4229d186b5e1b043b24e40236d4d262a4345f18a/src/components/NewTweetPopover/index.jsx#L35-L36